src/pages/success/index.tsx

```diff
@@ -21,14 +21,10 @@ export const SuccessPage = ({ finishOfframping, transactionId }: SuccessPageProp
     <main>
       <Box className="flex flex-col items-center justify-center mx-auto mt-12 ">
         <Checkmark />
-        <h1 className="mt-6 text-2xl font-bold text-center text-blue-700">Request made successfully</h1>
-        {transactionId && <TransactionInfo transactionId={transactionId} />}
-        <p className="mt-6 text-center">Normal processing times are between 5 and 10 minutes.</p>
+        <h1 className="mt-6 text-2xl font-bold text-center text-blue-700">All set! The withdrawal has been sent to your bank</h1>
         <div className="h-0.5 m-auto w-1/5 bg-pink-500 mt-8 mb-5" />
-        <p className="text-center text-gray-400">
-          If your transaction is not completed after 60 minutes please contact support on:
+        <p className="text-center text-gray-400"> Funds will received in 1 min (Instant SEPA) or 2 days (Standard SEPA)
```

Comment:
Yes SEPA is for european, we would have different text for Argentina

Reply:
But why are we adding this now if this is not with Argentina in mind? Mykobo is always using instant SEPA so we can also have the text only say 'Funds will be received in 1 min (Instant SEPA).' or 'Funds will arrive within 1 min (Instant SEPA)'.

Reply:
This is an improvement to existing text and also the user does not know that MYKOBO is using instant SEPA, for Argentina would have different text. I can understand this is not very DRY principle.

Reply:
I agree it's an improvement but if the user doesn't know that Mykobo is using instant SEPA, why not explicitly improve the improvement here? 😅 If we are mentioning SEPA which only applies to Europe and Mykobo is our only euro anchor, then it will always be instant SEPA and never standard SEPA.

Reply:
> why not explicitly improve the improvement here?

how would that be?

Reply:
@ebma Text updated, can we approve and merge this directly to production. This is priority ask as the current telegram link is incorrect on the production version.